_data/projects/openvpn-at-home.yml

```diff
@@ -0,0 +1,13 @@
+name: OpenVPN@Home
+desc: OpenVPN configuration management webapp
+site: https://github.com/ljvmiranda921/pyswarms
```

Comment:
@ezaquarii This is not the project that you are adding I guess.

Reply:
Oupsie! I'll fix it.